components/button-stateful/__docs__/storybook-stories.jsx

```diff
@@ -24,4 +24,7 @@ storiesOf(BUTTON_STATEFUL, module)
 	.add('Base', () => getButtonStateful())
 	.add('Disabled', () => getButtonStateful({ disabled: true }))
 	.add('Icon Text Button', () => <IconTextButton />)
-	.add('Icon Button', () => <IconButton />);
+	.add('Icon Button', () => <IconButton />)
+	.add('Base With aria-pressed', () =>
```

Comment:
This example is kind of confusing because `aria-pressed` is always true regardless of the state of the button.  Maybe use it as a controlled component so that this can be a more complete example?

Reply:
Yeah you are right.  I removed this example, and instead, extended the IconButton example to include the toggling of the aria attribute according to the state of the button.